packages/fxa-settings/src/components/App/gql.ts

```diff
@@ -108,3 +109,15 @@ export const GET_BACKUP_CODES_STATUS = gql`
     }
   }
 `;
+
+export const GET_RECOVERY_PHONE_AVAILABILITY = gql`
```

Comment:
This has merge conflicts with something Vijay added in the demo branch (this should be named `GET_RECOVERY_PHONE`, will change it when his PR is merged.